upload.go

```diff
@@ -82,6 +82,13 @@ func Configure() (*TarUploader, *Prefix, error) {
 		return nil, nil, errors.Wrapf(err, "Configure: failed to get AWS credentials; please specify AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY")
 	}
 
+	if endpoint := os.Getenv("AWS_ENDPOINT"); endpoint != "" {
+		config.Endpoint = aws.String(endpoint)
+	}
+
+	s3ForcePathStyle, _ := strconv.ParseBool(os.Getenv("AWS_S3_FORCE_PATH_STYLE"))
```

Comment:
I don't think so, the default for `config.S3ForcePathStyle` is false, `strconv.ParseBool` parses correctly: `true`, `false`, `1`, `0`. other than `true`  or `1` is false which is a good default.